CHANGES.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 0.4.1 (in progress)
+
+### Bug fixes
+
+-   always convert non-linear geometry types to linear geometry types (#140)
```

Comment:
On first read, I found the "non-linear" confusing (as I interpreted "linear" as LineString/LinearRing geometry types), so maybe add a "(for example, Curve, Surface, etc)" ?